jjb/mkck/mkck-trigger.yaml

```diff
@@ -31,8 +31,85 @@
     builders:
       - shell: |
           python sesci/convert-ctest-to-junit.py
-# End Macros
 
+- builder:
+    name: deliver_artifacts_sles
+    builders:
+      - multijob:
+          name: Deliver artifacts
+          #condition: ALWAYS
+          condition: SUCCESSFUL
+          projects:
+            - name: storage-deliver-mkck-artifact
+              current-parameters: true
+              abort-all-job: true
+              predefined-parameters: |
+                DIST_CEPH={dist_ceph}
+                SITE=ovh.net
+      - copyartifact:
+          project: storage-deliver-mkck-artifact
+          parameter-filters: 'DIST_CEPH={dist_ceph}'
+          filter: 'artifacts*.yaml'
+          optional: false
+          which-build: 'multijob-build'
+      - copyartifact:
+          project: storage-deliver-update-repos
+          parameter-filters: 'SES_VER={ses_ver},SITE=ovh.net'
+          filter: 'artifacts*.yaml'
+          optional: true
+          #target: ''
+          which-build: 'last-successful'
+      - shell: |
+          . v/bin/activate
+          . sesci/common/util
+          for i in artifacts*.yaml ; do
+            cat $i
+          done
+
+- builder:
+    # noop - we don't have artifacts for openSUSE Leap
+    name: deliver_artifacts_leap
+    builders:
+      - shell: |
+          /bin/true
```

Comment:
I wonder if it is really necessary, does it allow to just have empty 'builders' (I haven't check this)? I would avoid to sublimate noop with running remote `/bin/true`.

Reply:
You could just do following for noop:
```
- builder:
    name: deliver_artifacts_leap
    builders: []
```

Reply:
ah. Thanks for the hint. Done.